tests/unit_tests/test_sql_database_schema.py

```diff
@@ -70,7 +71,22 @@ def test_sql_database_run() -> None:
     stmt = insert(user).values(user_id=13, user_name="Harrison")
     with engine.begin() as conn:
         conn.execute(stmt)
-    db = SQLDatabase(engine, schema="schema_a")
+
+    with pytest.warns(Warning) as records:
```

Comment:
I'm out of the loop, but is duckdb stable enough to want to _assert_ that a warning is emitted? 

Or are we just trying to suppress the warning for our CI sanitation ?

If just the latter, do we want to do something like `@pytest.mark.filterwarnings("duckdb-engine doesn't..)` ?

Reply:
It's the latter, but I think we should keep it as is, so that it will fail once duckdb starts to support reflection which will allow us to remove the warning (my take on it)

Reply:
Merging as is, but let me know if you want to change